modules/certificate-injection-using-operators.adoc

```diff
@@ -10,6 +10,12 @@ Cluster Network Operator merges the user-provided and system CA certificates
 into a single bundle and injects the merged bundle into the Operator requesting
 the trust bundle injection.
 
+[IMPORTANT]
+====
+After adding a `config.openshift.io/inject-trusted-cabundle="true"` label to the config map, existing data in it is deleted. The Cluster Network Operator takes ownership of a config map and only accepts `ca-bundle` as data.
+It is recommended to use a separate config map to contain `service-ca.crt` instead of using the same config map that stores your pod configuration. You must not use `service-ca.crt` in the config map if a  `config.openshift.io/inject-trusted-cabundle="true"` label is added to the config map.
+====
```

Comment:
Hello! @sreber84 Does this combination look good in the above note? I tried to incorporate [your suggestion](https://github.com/openshift/openshift-docs/pull/58314#issuecomment-1498544167) and the existing note while trying to adhere to the style guide. Please let me know if something is missing.

Reply:
I'd change "It is recommended to use a separate config map to contain `service-ca.crt` instead of using the same config map that stores your pod configuration." with "It is required to use a separate config map to store the `service-ca.crt`  using `service.beta.openshift.io/inject-cabundle=true` annotation or other similiar configruation."

Also "You must not use `service-ca.crt` in the config map if a  `config.openshift.io/inject-trusted-cabundle="true"` label is added to the config map." should probably be phrased like "Using `config.openshift.io/inject-trusted-cabundle="true"` label and `service.beta.openshift.io/inject-cabundle=true` annotation on the same config map will trigger a problematic state and render the config map unusable"

Reply:
@xenolinux added some additional comments - hope those can be incooperated.

Reply:
@sreber84 Thanks!
 Incorporated the changes